tests/testthat/test-incidence.R

```diff
@@ -0,0 +1,128 @@
+test_that("graph_from_biadjacency_matrix() works -- dense", {
+  local_igraph_options(print.id = FALSE)
+  withr::local_seed(42)
+
+  inc <- matrix(sample(0:1, 15, repl = TRUE), 3, 5)
```

Comment:
@szhorvat any graph to recommend? the same example as in the other PR you've reviewed? Thank you :pray:

Reply:
Anything will work here.

To test `multiple=TRUE`, we need a matrix which have entries greater than 1 (e.g. 2 creates two edges).

A change is coming to this function in C/igraph 1.0. Right now, when `multiple=T`, non-integer values are rounded up. After 1.0, they will be truncated (i.e. rounded now). Generally, matrix elements must be integers with `multiple=T`. It is not impossible that we will start issuing warnings for non-integer values in the future.